README.md

```diff
@@ -12,6 +12,19 @@ Create a file named `.env` under the `frontend` folder and type the following in
 VUE_APP_BASE_PATH=/api
 ```
 
+Create a file named `.env` under the `backend` folder and type the following into it:
+
+**Note:** *Consider using mailtrap.io for testing mailing in your local environment.*
+
+```bash
+FRONTEND_BASE_PATH=http://stack.localhost
+
+EMAIL_HOST=smtp.mailtrap.io
+EMAIL_PORT=****
+EMAIL_USERNAME=*************
+EMAIL_PASSWORD=*************
```

Comment:
Hold environment variables in docker-compose

Remove env variable lines from readme

Remove dotenv files and pacakge

Take mailtrap.io line under an email configuration title

Reply:
done